contracts/user/UserManager.sol

```diff
@@ -769,14 +769,14 @@ contract UserManager is Controller, IUserManager, ReentrancyGuardUpgradeable {
 
         staker.stakedAmount -= amount.toUint96();
         staker.locked -= amount.toUint96();
-        staker.lastUpdated = (block.number).toUint64();
+        staker.lastUpdated = (getTimestamp()).toUint64();
 
         totalStaked -= amount;
 
         // update vouch trust amount
         vouch.trust -= amount.toUint96();
         vouch.locked -= amount.toUint96();
-        vouch.lastUpdated = (block.number).toUint64();
+        vouch.lastUpdated = (getTimestamp()).toUint64();
```

Comment:
We can probably cache this value to a variable, a few places where this is the case. I think it will be slightly more optimal.